### DISCUSSION TITLE
Datetime formatting

### DISCUSSION THREAD

**Discussion by ChristianBeilschmidt** [matheo/angular#14] [Q&A]
Is there an example of how to format the datetime within the input?

In the Angular Material examples they reference  `@angular/material-moment-adapter`, but this doesn't know anything about your newly introduced fields.

So I'm wondering if someone ever tried, for instance, to format it as `2021-03-27 17:35`?

**Comment by matheo** [comment]
Hi @ChristianBeilschmidt 
sorry for the late answer, I was super focused on [dynamic-forms](http://matheo.co/demos/dyn-forms) and didn't have enough brain available, now I have :) 

It seems that the NativeDateAdapter (the only one I'e ported so far) only supports the `hour12` parameter but cannot change the separator, it depends purely in the locale: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#using_options

So I've added a section [to the README file](https://github.com/matheo/angular/blob/f692c4f590e2f7529b36f9c08838765f07019834/libs/datepicker/README.md#date-formats-customization) to get this:
![image](https://user-images.githubusercontent.com/260185/112997483-c5ed7d00-9132-11eb-962a-39bd1dec3481.png)

I just added a new task to remind me to implement the `date-fns` adapter which is more robust supporting different Date Formats: https://github.com/matheo/angular/projects/3#card-58017806
